pinot-broker/src/main/java/com/linkedin/pinot/broker/routing/RoutingTableBuilderFactory.java

```diff
@@ -135,7 +138,7 @@ public RoutingTableBuilder createRoutingTableBuilder(TableConfig tableConfig) {
         }
         break;
     }
-    builder.init(_configuration, tableConfig, _propertyStore);
+    builder.init(_configuration, tableConfig, _propertyStore, _brokerMetrics);
```

Comment:
Seems the builder has been initiated twice, another one is in the caller side

Reply:
I will remove the init from here.